ES/ESmy004.xml

```diff
@@ -30,360 +30,665 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
-                                 <locus from="63r" to="70v"/> IX(8/fols. 63r-70v) </item>
-                              <item xml:id="q10">
+                                 <locus from="63r" to="70v"/>
+                               </item>
+                              <item xml:id="q10" n="10">
                                  <dim unit="leaf">8</dim>
-                                 <locus from="71r" to="78v"/> X(8/fols. 71r-78v) </item>
-                              <item xml:id="q11">
+                                 <locus from="71r" to="78v"/>
+                               </item>
+                              <item xml:id="q11" n="11">
                                  <dim unit="leaf">8</dim>
-                                 <locus from="79r" to="86v"/> XI(8/fols. 79r-86v) </item>
-                              <item xml:id="q12">
+                                 <locus from="79r" to="86v"/>
+                               </item>
+                              <item xml:id="q12" n="12">
                                  <dim unit="leaf">4</dim>
-                                 <locus from="87r" to="90v"/> XII(4/fols. 87r-90v) </item>
-                              <item xml:id="q13">
+                                 <locus from="87r" to="90v"/>
+                               </item>
+                              <item xml:id="q13" n="13">
                                  <dim unit="leaf">8</dim>
-                                 <locus from="91r" to="98v"/> XIII(8/fols. 91r-98v) </item>
-                              <item xml:id="q14">
+                                 <locus from="91r" to="98v"/>
+                               </item>
+                              <item xml:id="q14" n="14">
                                  <dim unit="leaf">8</dim>
-                                 <locus from="99r" to="106v"/> XIV(8/fols. 99r-106v) </item>
-                              <item xml:id="q15">
+                                 <locus from="99r" to="106v"/>
+                               </item>
+                              <item xml:id="q15" n="15">
                                  <dim unit="leaf">6</dim>
-                                 <locus from="107r" to="112v"/> XV(6/fols. 107r-112v) </item>
-                              <item xml:id="q16">
+                                 <locus from="107r" to="112v"/>
+                               </item>
+                              <item xml:id="q16" n="16">
                                  <dim unit="leaf">10</dim>
                                  <locus target="#10"/>
-                                 <locus from="113r" to="122v"/> XVI(10/fols. 113r-122v) </item>
-                              <item xml:id="q17">
+                                 <locus from="113r" to="122v"/>
+                               </item>
+                              <item xml:id="q17" n="17">
                                  <dim unit="leaf">7</dim>
```

Comment:
specify which one is single, or it won't work

Reply:
ok